File: javascript/src/models/portIoConfig.ts
// this file is @generated

export interface PortIoConfig {
  secret: string;
}

export const PortIoConfigSerializer = {
  _fromJsonObject(object: any): PortIoConfig {
    return {
      secret: object["secret"],
    };
  },

  _toJsonObject(self: PortIoConfig): any {
    return {
      secret: self.secret,
    };
  },
};

File: rust/src/models/port_io_config.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct PortIoConfig {
    pub secret: String,
}

impl PortIoConfig {
    pub fn new(secret: String) -> Self {
        Self { secret }
    }
}
